src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -248,6 +285,9 @@ class SubjectHeading extends React.Component {
             parentUuid={uuid}
             sortBy={sortBy}
             key={`${uuid}-list-${sortBy}`}
+            labelSorter={this.labelSorter}
```

Comment:
Hopefully we can figure out a better solution than passing the props this way :/ gets hard to manage.